hera / jet installations did not go well.

on jet there were `scotch@7.0.4` and `proj@8.1.0` issues:

```
==> Installing scotch-7.0.4-4avzhnbk2ocm7vkfgvv2hocnjeelr5nx
==> No binary for scotch-7.0.4-4avzhnbk2ocm7vkfgvv2hocnjeelr5nx found: installing from source
==> Cannot find version 7.0.4 in url_list
==> Error: FetchError: All fetchers failed for spack-stage-scotch-7.0.4-4avzhnbk2ocm7vkfgvv2hocnjeelr5nx
==> Installing libxau-1.0.8-3mroavg7o5fhr7yupniyc7rhvwaoff3i
==> No binary for libxau-1.0.8-3mroavg7o5fhr7yupniyc7rhvwaoff3i found: installing from source
==> Warning: Skipping build of ufs-weather-model-env-1.0.0-qdc3zfrlyef4hjvzb52w76avtiqzngut since scotch-7.0.4-4avzhnbk2ocm7vkfgvv2hocnjeelr5nx failed
```

```==> Installing proj-8.1.0-6g5mn2usqoioqtcfaqoy5vvfhiqcqeoa
==> No binary for proj-8.1.0-6g5mn2usqoioqtcfaqoy5vvfhiqcqeoa found: installing from source
==> Fetching file:///mnt/lfs4/HFIP/hfv3gfs/role.epic/spack-stack/source-cache/_source-cache/archive/22/22c5cdc5aa0832077b16c95ebeec748a0942811c1c3438c33d43c8d2ead59f48.tar.gz
==> Fetching file:///mnt/lfs4/HFIP/hfv3gfs/role.epic/spack-stack/source-cache/proj/proj-data-8.1.0.tar.gz
==> Error: ChecksumError: sha256 checksum failed for /mnt/lfs4/HFIP/hfv3gfs/role.epic/spack-stack/spack-stack-1.5.0/cache/build_stage/resource-proj-data-6g5mn2usqoioqtcfaqoy5vvfhiqcqeoa/proj-data-1.13.tar.gz
    Expected f1e5e42ba15426d01d1970be727af77ac9b88c472215497a5a433d0a16dd105b but got 76960d34d635aa127058ce654d89ea0eff91e2e4f2036482e677af5a88669b08. File size = 581536964 bytes. Contents = b'\x1f\x8b\x08\x00\xe3D\xef_\x00\x03\xd4\xb9c\x94&M...\x9f\xff\xfc\xe7\xff\xaf\xff\x016{\xd4+\x00V\xfa"'
```

on hera, issues with intel `netcdf-fortran@4.6.0` and, again,`proj-8.1.0` checksum errors (both compilers).

for netcdf, the error looks like:

```
libtool: warning: library '/apps/wgrib2/2.0.8/intel/18.0.3.222/lib/libnetcdf.la' was moved.
libtool: link: ar cr .libs/libnetcdf4_nf_interfaces.a .libs/module_netcdf4_nf_interfaces.o
libtool: link: ranlib .libs/libnetcdf4_f03.a
libtool: warning: library '/apps/wgrib2/2.0.8/intel/18.0.3.222/lib/libnetcdf.la' was moved.
ifort: command line warning #10434: option '-nofor_main' use with underscore is deprecated; use '-nofor-main' instead
ifort: command line warning #10006: ignoring unknown option '-mismatch_all'
ld: /apps/wgrib2/2.0.8/intel/18.0.3.222/lib/libnetcdf.a(attr.o): relocation R_X86_64_32S against `.rodata' can not be used when making a shared object; recompile with -fPIC
ld: /apps/wgrib2/2.0.8/intel/18.0.3.222/lib/libnetcdf.a(putget.o): relocation R_X86_64_32S against `.rodata' can not be used when making a shared object; recompile with -fPIC
ld: /apps/wgrib2/2.0.8/intel/18.0.3.222/lib/libnetcdf.a(error.o): relocation R_X86_64_32 against `unknown' can not be used when making a shared object; recompile with -fPIC
ld: /apps/wgrib2/2.0.8/intel/18.0.3.222/lib/libnetcdf.a(nc.o): relocation R_X86_64_32S against `.rodata' can not be used when making a shared object; recompile with -fPIC
ld: final link failed: Nonrepresentable section on output
make[2]: *** [libnetcdff.la] Error 1
make[1]: *** [all] Error 2
make[1]: Leaving directory `/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.5.0/cache/build_stage/spack-stage-netcdf-fortran-4.6.0-z5pu7eld2j7cftvrdo6vp2mudy5poabq/spack-src/fortran'
make: *** [all-recursive] Error 1
==> Error: ProcessError: Command exited with status 2:
    'make' '-j6' 'V=1'

4 errors found in build log:
     532    ld: /apps/wgrib2/2.0.8/intel/18.0.3.222/lib/libnetcdf.a(nc.o): relocation R_X86_64_32S against `.rodata' can not be used when making a shar
            ed object; recompile with -fPIC
     533    ld: /apps/wgrib2/2.0.8/intel/18.0.3.222/lib/libnetcdf.a(string.o): relocation R_X86_64_32 against `.rodata.str1.4' can not be used when mak
            ing a shared object; recompile with -fPIC
     534    ld: /apps/wgrib2/2.0.8/intel/18.0.3.222/lib/libnetcdf.a(v1hpg.o): relocation R_X86_64_32 against `.rodata.str1.4' can not be used when maki
            ng a shared object; recompile with -fPIC
     535    ld: /apps/wgrib2/2.0.8/intel/18.0.3.222/lib/libnetcdf.a(var.o): relocation R_X86_64_32 against `.rodata.str1.4' can not be used when making
             a shared object; recompile with -fPIC
     536    ld: /apps/wgrib2/2.0.8/intel/18.0.3.222/lib/libnetcdf.a(utf8proc.o): relocation R_X86_64_32S against symbol `utf8proc_utf8class' can not be
             used when making a shared object; recompile with -fPIC
     537    ld: /apps/wgrib2/2.0.8/intel/18.0.3.222/lib/libnetcdf.a(posixio.o): relocation R_X86_64_32 against `.rodata.str1.4' can not be used when ma
            king a shared object; recompile with -fPIC
  >> 538    ld: final link failed: Nonrepresentable section on output
  >> 539    make[2]: *** [libnetcdff.la] Error 1
     540    make[2]: Leaving directory `/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.5.0/cache/build_stage/spack-stage-netcdf-fortran-4.6
            .0-z5pu7eld2j7cftvrdo6vp2mudy5poabq/spack-src/fortran'
  >> 541    make[1]: *** [all] Error 2
     542    make[1]: Leaving directory `/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.5.0/cache/build_stage/spack-stage-netcdf-fortran-4.6
            .0-z5pu7eld2j7cftvrdo6vp2mudy5poabq/spack-src/fortran'
  >> 543    make: *** [all-recursive] Error 1

See build log for details:
  /scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.5.0/cache/build_stage/spack-stage-netcdf-fortran-4.6.0-z5pu7eld2j7cftvrdo6vp2mudy5poabq/spack-build-out.txt
```

--- a/doc/source/PreConfiguredSites.rst
+++ b/doc/source/PreConfiguredSites.rst
@@ -36,9 +36,9 @@ Ready-to-use spack-stack 1.5.0 installations are available on the following, ful
 |                     +----------------------------------+-----------------+---------------------------------------------------------------------------------------------------------+-------------------------------+
 |                     | Gaea C5                          | Intel           | ``/lustre/f2/dev/wpo/role.epic/contrib/spack-stack/c5/spack-stack-dev-20230717/envs/unified-env``       | Dom Heinzeller / ???          |
 | NOAA (RDHPCS)       +----------------------------------+-----------------+---------------------------------------------------------------------------------------------------------+-------------------------------+
-|                     | Hera                             | GCC, Intel      | ``/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.4.1/envs/unified-env``                     | Mark Potts / Dom Heinzeller   |
+|                     | Hera                             | GCC, Intel      | ``/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.5.0/envs/unified-env``                     | Mark Potts / Dom Heinzeller   |
 |                     +----------------------------------+-----------------+---------------------------------------------------------------------------------------------------------+-------------------------------+
-|                     | Jet                              | GCC, Intel      | ``/mnt/lfs4/HFIP/hfv3gfs/role.epic/spack-stack/spack-stack-1.4.1/envs/unified-env``                     | Cam Book / Dom Heinzeller     |
+|                     | Jet                              | GCC, Intel      | ``/mnt/lfs4/HFIP/hfv3gfs/role.epic/spack-stack/spack-stack-1.5.0/envs/unified-env``                     | Cam Book / Dom Heinzeller     |
 +---------------------+----------------------------------+-----------------+---------------------------------------------------------------------------------------------------------+-------------------------------+
 |                     | Narwhal                          | Intel           | ``/p/app/projects/NEPTUNE/spack-stack/spack-stack-1.5.0/envs/unified-env-intel-2021.4.0``               | Dom Heinzeller / Sarah King   |
 |                     +----------------------------------+-----------------+---------------------------------------------------------------------------------------------------------+-------------------------------+
@@ -537,24 +537,24 @@ The following is required for building new spack environments and for using spac
    module load ecflow/5.5.3
    module load mysql/8.0.31
 
-For ``spack-stack-1.4.1`` with Intel, load the following modules after loading miniconda and ecflow:
+For ``spack-stack-1.5.0`` with Intel, load the following modules after loading miniconda and ecflow:
 
 .. code-block:: console
 
-   module use /scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.4.1/envs/unified-env/install/modulefiles/Core
+   module use /scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.5.0/envs/unified-env/install/modulefiles/Core
    module load stack-intel/2021.5.0
    module load stack-intel-oneapi-mpi/2021.5.1
-   module load stack-python/3.9.12
+   module load stack-python/3.10.8
    module available
 
-For ``spack-stack-1.4.1`` with GNU, load the following modules after loading miniconda and ecflow:
+For ``spack-stack-1.5.0`` with GNU, load the following modules after loading miniconda and ecflow:
 
 .. code-block:: console
 
-   module use /scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.4.0/envs/unified-env/install/modulefiles/Core
+   module use /scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.5.0/envs/unified-env/install/modulefiles/Core
    module load stack-gcc/9.2.0
    module load stack-openmpi/4.1.5
-   module load stack-python/3.9.12
+   module load stack-python/3.10.8
    module available
 
 Note that on Hera, a dedicated node exists for ``ecflow`` server jobs (``hecflow01``). Users starting ``ecflow_server`` on the regular login nodes will see their servers being killed every few minutes, and may be barred from accessing the system.
@@ -576,24 +576,24 @@ The following is required for building new spack environments and for using spac
    module use /lfs4/HFIP/hfv3gfs/role.epic/modulefiles
    module load mysql/8.0.31
 
-For ``spack-stack-1.4.1`` with Intel, load the following modules after loading miniconda and ecflow:
+For ``spack-stack-1.5.0`` with Intel, load the following modules after loading miniconda and ecflow:
 
 .. code-block:: console
 
-   module use /mnt/lfs4/HFIP/hfv3gfs/role.epic/spack-stack/spack-stack-1.4.1/envs/unified-env/install/modulefiles/Core
+   module use /mnt/lfs4/HFIP/hfv3gfs/role.epic/spack-stack/spack-stack-1.5.0/envs/unified-env/install/modulefiles/Core
    module load stack-intel/2021.5.0
    module load stack-intel-oneapi-mpi/2021.5.1
-   module load stack-python/3.9.12
+   module load stack-python/3.10.8
    module available
 
-For ``spack-stack-1.4.1`` with GNU, load the following modules after loading miniconda and ecflow:
+For ``spack-stack-1.5.0`` with GNU, load the following modules after loading miniconda and ecflow:
 
 .. code-block:: console
 
-   module use /mnt/lfs4/HFIP/hfv3gfs/role.epic/spack-stack/spack-stack-1.4.1/envs/unified-env/install/modulefiles/Core
+   module use /mnt/lfs4/HFIP/hfv3gfs/role.epic/spack-stack/spack-stack-1.5.0/envs/unified-env/install/modulefiles/Core
    module load stack-gcc/9.2.0
    module load stack-openmpi/3.1.4
-   module load stack-python/3.9.12
+   module load stack-python/3.10.8
    module available
 
 ------------------------------

